go.mod

```diff
@@ -3,13 +3,17 @@ module github.com/optimizely/go-sdk
 go 1.12
 
 require (
+	github.com/DATA-DOG/godog v0.7.13
```

Comment:
Can you remove these please? They are used only for FSC testing, which shoudn't be included in our main deps